### DISCUSSION TITLE
会议时间调整

### DISCUSSION THREAD

**Discussion by stevending1st** [kaiyuanshe/RFC#6] [Ideas]
项目委员会例会与 ONES 例会冲突
  - 方案一：项目委员会例会改为双周一次，与 ONES 例会交叉；
  - 方案二：「项目委员会例会」或「 ONES 例会」调整时间。 

[周信息汇总](https://kaiyuanshe.feishu.cn/base/bascniBToawwx7cWIupv1Ry129g)

最新进度：

「 ONES 例会」调整为每双周周五晚上 21:30-22:00

但是 **项目委员会例会双周化仍然需要推进**

**Comment by TechQuery** [comment]
> 项目委员会例会与 ONES 例会冲突
> - 方案一：项目委员会例会改为双周一次，与 ONES 例会交叉；
> - 方案二：「项目委员会例会」或「 ONES 例会」调整时间。

ONES 组长对我给 ONES 提的多条建议视而不见、已读不回，今年执委会 ONES 组也从未做过工作汇报。我们一个活跃部门为什么要为不活跃部门的成员不好参会而操心？

![Screenshot_2023-10-04-22-48-39-312_com.ss.android.lark-edit.jpg](https://github.com/kaiyuanshe/RFC/assets/19969570/dcc3c5c0-0164-49e4-839a-6549d59e13b4)
